packages/scrypt-kdf/scrypt-kdf.1.2.0/opam

```diff
@@ -0,0 +1,35 @@
+opam-version: "2.0"
+synopsis: "The scrypt Password-Based Key Derivation Function"
+description: """
+A pure OCaml implementation of [scrypt](https://en.wikipedia.org/wiki/Scrypt) password based key derivation function,
+as defined in [The scrypt Password-Based Key Derivation Function internet draft](https://tools.ietf.org/html/draft-josefsson-scrypt-kdf-04),
+including test cases from the RFC.
+"""
+maintainer: ["Alfredo Beaumont <alfredo.beaumont@gmail.com>"]
+authors: ["Alfredo Beaumont <alfredo.beaumont@gmail.com>" "Sonia Meruelo <smeruelo@gmail.com>"]
+license: "BSD-2-Clause"
+homepage: "https://github.com/abeaumont/ocaml-scrypt-kdf"
+bug-reports: "https://github.com/abeaumont/ocaml-scrypt-kdf/issues"
+dev-repo: "git+https://github.com/abeaumont/ocaml-scrypt-kdf.git"
+doc: "https://abeaumont.github.io/ocaml-scrypt-kdf/"
+depends: [
+  "ocaml" {>= "4.07.0"}
+  "dune" {>= "1.8.0"}
+  "cstruct" {>= "6.0.0"}
+  "mirage-crypto"
+  "pbkdf" {>= "0.1.0"}
+  "salsa20-core" {>= "0.1.0"}
```

Comment:
```suggestion
  "salsa20-core" {>= "1.0.0"}
```

looks like some older OCaml releases need a newer version of salsa-core that uses dune to link against the C library